### ISSUE TITLE
workers are not visible in resque-web

### ISSUE THREAD

**Issue by mguterl** [staugaard/resque-multi-job-forks#6]
We added resque-mutli-job-forks to our project today, however, it appears that the workers disappear from the resque-web UI.

When the workers first start they show up in the UI, but after the fork for the second time, they all start to appear to die off.  Looking at top the workers are obviously working and the jobs are being processed in the queue.

We are using resque 1.10.0

**Comment by Narnach** [comment]
We have the same problem using resque 1.19.0 on rails 3.1.3 using the latest git version of resque-multi-job-forks.

Did you manage to find a solution, @mguterl ?

**Comment by mguterl** [comment]
@Narnach I don't believe we found any solution, we're still using an older version of Resque but plan on upgrading soon.

**Comment by Narnach** [comment]
I just tested the commits by @mguterl and the work on top of that by @dolores and together they seem to solve the problem, at least when doing simple tests in development mode.

The relevant commits are: recruitmilitary/resque-multi-job-forks@93b4f6c994994c1d462f42ee903d4444a6709fea and dolores/resque-multi-job-forks@547935e1c60b65b90e1c5c2093833b191002fd7a
